docs/content/preview/reference/configuration/yugabyted.md

```diff
@@ -762,7 +762,7 @@ Create a local single-node universe with IPv6 address:
 Create a single-node locally and join other nodes that are part of the same universe:
```

Comment:
Check for other pages where multi-node universes are created using join where this can be clarified - eg the coredb manual deploy pages

Applies to previous versions?